code/game/machinery/transformer.dm

```diff
@@ -98,7 +98,6 @@
 	if(masterAI)
 		R.connected_ai = masterAI
 		R.lawupdate = TRUE
-		R.lawsync()
```

Comment:
Same here.

Reply:
This is still called twice, but `post_lawchange()` prevents duplicate calling of `show_laws()` within the same tick which is all that matters to the user.

Reply:
No longer called twice.